scripts/enabled_test_modules.py

```diff
@@ -227,6 +227,16 @@
         "Cannot resolve keyword 'porcupine' into field",
         'Argument 1 to "set" of "RelatedManager" has incompatible type "int"',
     ],
+    'mail': [
+        'List item 1 has incompatible type "None"; expected "str"',
+        'Incompatible return value type (got "None", expected "bool")',
+        'Argument 1 to "push" of "SMTPChannel" has incompatible type "str"; expected "bytes"',
+        'Value of type "Union[List[Message], str, bytes, None]" is not indexable',
+        'Incompatible types in assignment (expression has type "bool", variable has type "Union[SMTP_SSL, SMTP, None]")',
```

Comment:
ignore for https://github.com/django/django/blob/master/tests/mail/tests.py#L1450